test/integration/view_event_test.rb

```diff
@@ -51,8 +54,9 @@ class ViewEventTest < ActionDispatch::IntegrationTest
   end
 
   test 'finds events for search term' do
-    get "/#{@conference.acronym}/events?q%5Bs%5D=track_name+asc&term=frap&utf8=%E2%9C%93"
+    get "/#{@conference.acronym}/events?q%5Bs%5D=track_name+asc&term=#{@conference.events.last.title.split.last}&utf8=%E2%9C%93"
     assert_response :success
     assert_includes @response.body, 'frap'
```

Comment:
This assert is not guaranteed anymore, once you don't use the search term 'frap' anymore. Or am I missing something?

Reply:
The events created by this test are `Introducing frap part x`  - see https://github.com/frab/frab/blob/master/test/factories/event.rb#L2

So I filter by the last word (the serial number) and we'll see the title `Introducing frap part x` for one event anyway.